src/index.tsx

```diff
@@ -112,6 +114,9 @@ class RichMarkdownEditor extends React.PureComponent<Props, State> {
     onClickLink: href => {
       window.open(href, "_blank");
     },
+    onKeyDown: () => {
```

Comment:
I think we can just define it as an optional prop in the types instead.

Reply:
Updated!